css/screens/login-signup.css

```
@@ -0,0 +1,122 @@
+.login__wrap {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  width: 640px;
+}
+
+.main__wrap {
+  margin-top: 40px;
+}
+
+.login-form {
+  display: flex;
+  flex-direction: column;
+}
+
+.login-form__label {
+  margin-bottom: 16px;
+  font-size: 18px;
+  font-weight: 700;
+  color: var(--Secondary-800);
+  /* pretendard/2lg-18px-bold */
+  line-height: 1.444;
+}
+
+.login-form__input {
+  display: flex;
+  align-items: center;
+  width: 640px;
+  height: 56px;
+  margin-bottom: 24px;
+  padding: 16px 24px;
+  border: none;
+  border-radius: 12px;
+  background: var(--Secondary-100);
+}
+
+.login-form__input:focus {
+  outline: 1px solid #3692ff;
+}
+
+.login-form__input[type="placeholder"] {
+  font-size: 16px;
+  font-weight: 400;
+  color: var(--Secondary-400);
+  /* pretendard/lg-16px-regular */
+  line-height: 1.625;
+}
+
+.login-form__btn {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 640px;
+  height: 56px;
+  padding: 16px 124px;
+  border: none;
+  border-radius: 40px;
+  background: var(--Secondary-400);
+  font-size: 20px;
+  font-weight: 600;
+  text-align: center;
+  color: var(--Cool-Gray-100, #f3f4f6);
```

P1:
해당 변수 variables.css 파일에 선언이 안 되어 있는것 같습니다.
아래처럼 사용해주시거나 아니면 선언 후 사용해주세요~

```suggestion
  color:  #f3f4f6;
```